### ISSUE TITLE
Clarification on h264 error and attention mask warning

### ISSUE THREAD

**Issue by arsoyul** [HYUNJS/STTM#3]
Hi,
Thank you for sharing this great work. I'm currently working on reproducing the experiments in your paper and have successfully obtained similar results, which is very exciting.

During the process, I've encountered a couple of minor warnings/errors. And I'd like to ask about ensuring my setup is correct.

1. h264 mmco: unref short failure error
When running the feature extraction script, this error appears for some datasets. But the process was done well, including evaluation.
Error : [h264 @ 0x55b67d5f7540] mmco: unref short failure
Is this non-critical error related to specific video encodings that can be safely ignored, or does it indicate a potential problem with my dataset preparation or environment?

2. attention_mask warning
In model evaluation, the following warning is displayed.
Warn : The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
The model seems to be running correctly. Is it safe to assume this is an informational warning that can be ignored?

**Comment by HYUNJS** [comment]
Thanks for your interest!

Your questions about the warning messages are negligible.

1. I have been faced h264 warning message for a long time, even before this project. However, it does not cause serious problems, so I ignored it. I suspect the downloaded video files might be corrupted in terms of metadata. Just in case, please check whether the video files play correctly on your side.
2. When using FlashAttention, the pre-computed attention mask is not actually used as far as I know for transformers library version I used. Thus, this warning message can be ignored.

When you use the "--reproduce" config, I expect you to obtain the same accuracy results. I have verified this across different servers (but all using the same A100 GPUs). Please let me know if you encounter any issues!
